gradle/dependencies.gradle

```diff
@@ -57,7 +57,8 @@ def misc = [
 
 def rx = [
     android: 'io.reactivex.rxjava2:rxandroid:2.0.1',
-    java: 'io.reactivex.rxjava2:rxjava:2.1.0'
+    java: 'io.reactivex.rxjava2:rxjava:2.1.0',
+    lifecycle: 'com.trello.rxlifecycle2:rxlifecycle:2.2.0'
```

Comment:
this should be separate, this block is just for reactivex. Probably under misc

Reply:
Fixed top-level comments - renaming and dependencies.gradle in  https://github.com/uber/AutoDispose/pull/118/commits/cbf3e10fed2ae7f177a35cf4206626fb878956d2